website/docs/configuration/ort-yml.md

```diff
@@ -6,7 +6,9 @@ increased degree of automation and local configurations should only be done if t
 
 * [excludes](#excludes) - Mark [files, directories](#excluding-paths) or [package manager scopes](#excluding-scopes) as
   not included in released artifacts.
-* [curations](#curations) - Overwrite package metadata, set a concluded license or correct license findings.
+* [curations](#curations) - Override package metadata, set a concluded license or correct license findings. Needs to be
+  enabled in global ORT configuration ([config.yml](../getting-started/usage.md#ort-configuration-file)) via
+  `enableRepositoryPackageCurations: true`.
```

Comment:
Those admonitions look nice, but I'm a bit concerned that they don't render correctly when indented in a list. Is there a way to preview this? Neither IntelliJ nor GitHub seem to be able to understand this syntax.

Reply:
My suggestion was not to add the admonition to the list, but to add it to the beginning of the curations section. Unfortunately I don't know a way to preview this except for running the site locally.

Reply:
> My suggestion was not to add the admonition to the list, but to add it to the beginning of the curations section.

Hmm, I'm not sure I like that, if I'm understanding correctly what you mean. As enabling is only needed for this specific feature, I believe that note should stay here. And I don't want to repeat too much of that note at the section level either.

Does Docuaurus also support GitHub-style admonition syntax like

```
> **Note**
> Some text
```

which renders like

> **Note**
> Some text

?

Reply:
> Does Docuaurus also support GitHub-style admonition syntax like

Not (yet), it seems: https://github.com/facebook/docusaurus/issues/7471

Reply:
In any case, I'd like to avoid syntax that's not valid in both (GitHub flavored) Markdown and Docusaurus. Just in case we ever switch so something else again, I'd like to avoid running into a "vendor lock-in".

Reply:
> Hmm, I'm not sure I like that, if I'm understanding correctly what you mean. As enabling is only needed for this specific feature, I believe that note should stay here. And I don't want to repeat too much of that note at the section level either.

I mean to put the admonition at the beginning of this section which describes the usage of curations in the `.ort.yml`. In the list at the beginning I think it's very easy to miss that important information, especially when navigating via the TOC on the right side: https://oss-review-toolkit.org/ort/docs/configuration/ort-yml#curations

> In any case, I'd like to avoid syntax that's not valid in both (GitHub flavored) Markdown and Docusaurus. Just in case we ever switch so something else again, I'd like to avoid running into a "vendor lock-in".

I wouldn't see this as a problematic vendor lock-in, it's easy to switch to another syntax if necessary and I don't expect that we will use too many admonitions. However, I tried [this remark plugin](https://github.com/myl7/remark-github-beta-blockquote-admonitions) to enable the GitHub syntax and I could get it to compile with [this hint](https://github.com/facebook/docusaurus/issues/5379#issuecomment-1525220908), but for some reason it didn't work. It's probably worth to wait a bit until Docusaurus 3 is released and then try again after the upgrade.

Reply:
> I mean to put the admonition at the beginning of this section which describes the usage of curations in the `.ort.yml`.

Ok, I hope to now have it like you suggest.